docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -4,89 +4,92 @@
 
 beta::[]
 
-{anomaly-detect-cap} alerts run scheduled checks on an {anomaly-job} or a group 
-of jobs to detect anomalies with certain conditions. If an anomaly meets the 
-conditions, the alert triggers the defined action. For example, you can create 
-an alert that checks an {anomaly-job} every fifteen minutes for critical 
-anomalies and notifies you in an email. This page helps you to configure an 
-{anomaly-detect} alert. To learn more about alerts in the {stack}, refer to 
+{anomaly-detect-cap} alerting runs scheduled checks on an {anomaly-job} or a 
+group of jobs to detect anomalies with certain conditions. If an anomaly meets 
+the rules you defined, an alert is created and the associated action is 
+triggered. For example, you can create a rule to check an {anomaly-job} every 
+fifteen minutes for critical anomalies and to notify you in an email. This page 
+helps you to configure {anomaly-detect} alerting. To learn more about alerting 
+in the {stack}, refer to 
 {kibana-ref}/alerting-getting-started.html#alerting-getting-started[Alerting and Actions].
```

Comment:
I don't know if we're in control of all links, but in the preview, I see `general alert details.`, but it should be `general rule details.`